yjit/src/invariants.rs

```diff
@@ -33,7 +28,10 @@ pub struct Invariants {
     /// Tracks block assumptions about method lookup. Maps a class to a table of
     /// method ID points to a set of blocks. While a block `b` is in the table,
     /// b->callee_cme == rb_callable_method_entry(klass, mid).
-    method_lookup: HashMap<VALUE, HashMap<ID, Vec<MethodLookupDependency>>>
+    method_lookup: HashMap<VALUE, HashMap<ID, Vec<(BlockRef, ID)>>>,
+
+    /// Tracks block assumptions about running in single ractor mode.
+    single_ractor: Vec<BlockRef>
```

Comment:
I feel like the names and comments for these fields could be more descriptive. It's unclear to me reading this comment what `method_lookup` is or what it's used for 🤔 

Previously `single_ractor` was named `blocks_assuming_single_ractor` and that was a super clear name. Maybe the comment could make it clearer, because otherwise, what are block assumptions about running in single ractor mode?

Reply:
Updated!